smarts/core/opendrive_road_network.py

```diff
@@ -226,6 +226,11 @@ def from_spec(
         cls,
         map_spec: MapSpec,
     ):
+        if map_spec.shift_to_origin:
+            logger = logging.getLogger(cls.__name__)
+            logger.warning(
+                "OpenDrive road networks do not yet support the 'shift_to_origin' option."
+            )
```

Comment:
Unlike `SUMO`, `OpenDRIVE` doesn't require shifting of origin, since offset computation are fairly more accurate in `OpenDRIVE`.

Reply:
understood.  But I think the warning is still needed here just in case some user tries to use `shift_to_origin` for some other reason of their own.